Providing an example of button ui component (#5326)
magento/devdocs: guides/v2.2/ui_comp_guide/components/ui-button.md

`actionName` | Name of the component's method to be invoked. | String | Required |
`params` | A list of arguments that will be passed to the method. | Array | Optional |

## Examples

### Configure component

> Here is an example of how Button component integrates with Listing component:

Reviewer: ```suggestion
Here is an example of a Button component ("Test Button") added below a Listing component:
```

```xml
...
<button name="my_new_button">
    <argument name="data" xsi:type="array">
        <item name="config" xsi:type="array">
            <item name="actions" xsi:type="array">
                <!-- Add your button's actions here -->
                <item name="0" xsi:type="array">
                    <item name="targetName" xsi:type="string">TARGET_NAME</item>
                    <item name="actionName" xsi:type="string">ACTION_NAME</item>
                </item>
            </item>
        </item>
    </argument>
    <settings>
        <displayAsLink>false</displayAsLink>
        <title><![CDATA[Test Button]]></title>
    </settings>
</button>
...
```

### Result

> The result is a new added button to page listing.

Reviewer: ```suggestion
The Button component appears below a Listing component on the page, as follows:
```
